firefox_puppeteer/ui/windows.py

```diff
@@ -186,6 +181,16 @@ def switch_to(self, target):
 
         return self.create_window_instance(target_handle)
 
+    @classmethod
+    def register_window(cls, window_type, window_constructor):
+        """Registers a chrome window with this class so that this class may in
+        turn create the appropriate window instance later on.
+
+        :param window_type: The type of window.
+        :param window_constructor: The constructor of the window
```

Comment:
nit: It's the class we pass in here and not the constructor. So call it `window_class`.